examples/common-chunk-grandchildren/webpack.config.js

```diff
@@ -0,0 +1,15 @@
+var webpack = require("../../");
+
+module.exports = {
+	entry: {
+		main: ["./example.js"]
+	},
+	plugins: [
+		new webpack.optimize.CommonsChunkPlugin({
+			name: "main",
+			minChunks: 2,
+			children: true,
+			deepChildren: true,
+		})
+	]
+};
```

Comment:
Could you also a `async` mode to this example? You can pass multiple configurations to include both modes in a single example.